src/app/dashboard/sensor-widget/sensor-widget.service.js

```diff
@@ -0,0 +1,28 @@
+import {Injectable} from 'angular2/core';
+import io from 'socket.io-client';
+
+@Injectable()
+export default class SensorWidgetService {
+    constructor() {
+        this.io = io;
+        this.setupSocket();
+    }
+
+    setupSocket() {
+        // TODO add dev config via webpack plugin to manage proper ws host and port
+        this.socket = this.io('ws://localhost:3000');    // real backEnd
```

Comment:
!fix
we already got config file for http operations
https://github.com/garage-it/SmartHouse-frontend/blob/master/src/app/sh-http/sh-http.config.js
imho, it should be joined into one config file and taken from there.
no additional logic, just importing from a particular folder -- `prod.config.js` and `index.js`, for example
later we could enrich the `index.js` with logic of picking config